### PR TITLE
Remove 8 apps & add 12 apps to Chinese list & fix bugs

### PR DESCRIPTION
**Pull request by Qifull**
- Removed: iFFmpeg, Soda Player, Sonora, 零信, 今目标, RTX_腾讯通, Yomail, AlloyDesktop, Adium Messenger
- Added: 1.1.1.1, escrcpy, PopClip, MonitorControl, Mastodon, Signal, sing-box, Matrix, 企业微信, Microsoft Teams, Chrome Remote Desktop, lima-vm, PDFgear to Chinese list
- Fixed various bugs

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "githubPullRequests.ignoredPullRequestBranches": [
+        "master"
+    ]
+}
```
Comment (jaywcjlove) [inline .vscode/settings.json:5]: @Qifull This VSCode configuration is necessary to add, isn’t it?

File: README-zh.md [hunk 2]
```diff
@@ -165,7 +166,6 @@ Awesome Mac
 * [Cacher](https://www.cacher.io/) - 基于云的团队代码片段管理器，具有Gist同步，VSCode/Atom/Sublime软件包和Mac/Windows/Linux/Web客户端。
 * [CubicBezier](https://github.com/isaced/CubicBezier) - 一个适用于 macOS 的贝塞尔曲线生成器。 [![Open-Source Software][OSS Icon]](https://github.com/isaced/CubicBezier) ![Freeware][Freeware Icon] [![App Store][app-store Icon]](https://itunes.apple.com/us/app/cubicbezier/id1228492117?mt=12)
 * [DevHub](https://wangchujiang.com/DevHub/) - 一个功能丰富的离线应用程序，精心制作，以支持开发人员在他们的日常任务。![Freeware][Freeware Icon] [![App Store][app-store Icon]](https://apps.apple.com/app/devhub/id6476452351)
-* [DevToys](https://github.com/DevToys-app/DevToys) - 开发者的瑞士军刀——一款帮助开发者处理日常任务的桌面应用。[![Open-Source Software][OSS Icon] ![Freeware][Freeware Icon]](https://github.com/DevToys-app/DevToys)
```
Comment (jaywcjlove) [inline README-zh.md:168]: image

@Qifull Why was it deleted?

File: README-zh.md [hunk 3]
```diff
@@ -461,44 +458,48 @@ Awesome Mac
...
 * [Keybase](https://keybase.io/) - 一个安全的消息应用程序! [![Open-Source Software][OSS Icon]](https://github.com/keybase) ![Freeware][Freeware Icon]
 * [Maipo脉搏](http://weiboformac.sinaapp.com/) - 微博第三方 Mac 应用。![Freeware][Freeware Icon] [![App Store][app-store Icon]](https://itunes.apple.com/cn/app/weibox/id789066512)
-* [Messenger](https://fbmacmessenger.rsms.me/) - Facebook 第三方聊天工具。[![Open-Source Software][OSS Icon] ![Freeware][Freeware Icon]](https://github.com/rsms/fb-mac-messenger)
+* [Mastodon](https://mastodon.social/) - 一个自由开源的去中心化的分布式微博客社交网络。[![Freeware][Freeware Icon]](https://joinmastodon.org/zh/apps) [![Open-Source Software][OSS Icon]](https://github.com/tootsuite/mastodon)
+* [Matrix](https://matrix.org/) - 一个用于安全、去中心化通信的开放网络。![Freeware][Freeware Icon] ![Open-Source Software][OSS Icon]
 * [QQ](http://im.qq.com/macqq/index.shtml) - QQ for Mac App。![Freeware][Freeware Icon]
 * [Rambox](http://rambox.pro/) - 消息和电子邮件应用程序，将常见的Web应用程序组合成一个程序。 [![Open-Source Software][OSS Icon]](https://github.com/saenzramiro/rambox) ![Freeware][Freeware Icon]
+* [Signal](https://signal.org/) - 畅所欲言
+来尝试一种别样的通讯体验吧。Signal 不仅提供各种常见的功能，还尤为注重隐私安全。
+![Freeware][Freeware Icon]
```
Comment (jaywcjlove) [inline README-zh.md:488]: @Qifull  There is no need for a line break here.

File: .vscode/settings.json (hunk 1 above)
Comment (Qifull) [reply .vscode/settings.json:5]: Sorry, my mistake. Made so many changes at once, didn’t have time to react.

File: README-zh.md (hunk 2 above)
Comment (Qifull) [reply README-zh.md:168]: **Deleted by accident, likely during the merge. This is conflict-related and can be re-added.**

File: README-zh.md (hunk 3 above)
Comment (Qifull) [reply README-zh.md:488]: I see

File: README-zh.md [hunk 4]
```diff
@@ -461,44 +459,46 @@ Awesome Mac
...
 * [Multipass](https://multipass.run/) - 适用于任何工作站的 Ubuntu 虚拟机。 [![Open-Source Software][OSS Icon]](https://github.com/canonical/multipass)
+* [OrbStack](https://orbstack.dev/) - 在 Mac 上无缝高效地运行 Docker 和 Linux。Docker Desktop 替代品，可帮助您更快地工作。![Freeware][Freeware Icon]
 * [Parallels Desktop](http://www.parallels.com/) - 虽然好用但是收费机制，更新花钱、花钱、花钱。
 * [Portainer](https://portainer.io/) - 基于网页管理 Docker 容器和 swarm 集群。 [![Open-Source Software][OSS Icon]](https://github.com/portainer/portainer) ![Freeware][Freeware Icon]
 * [UTM](https://mac.getutm.app/) - 适用于 iOS 和 macOS 的全功能系统模拟器和虚拟机主机。[![Open-Source Software][OSS Icon]](https://github.com/utmapp/UTM)
+* [Veertu](https://veertu.com) - Mac 上轻量级的虚拟机。通过一种高响应，沙箱且本地化的方式在你在 Mac 上运行虚拟机。![Freeware][Freeware Icon]
 * [Virtual Box](http://www.virtualbox.org) - 免费，带 NTFS 读写，不用买 ParagonNTFS，省100块。![Freeware][Freeware Icon]
 * [VMware Fusion Pro](http://www.vmware.com/) - 功能强大的虚拟机软件，个人使用免费，企业需要许可![Freeware][Freeware Icon]
-* [Veertu](https://veertu.com) - Mac 上轻量级的虚拟机。通过一种高响应，沙箱且本地化的方式在你在 Mac 上运行虚拟机。![Freeware][Freeware Icon]
-* [OrbStack](https://orbstack.dev/) - 在 Mac 上无缝高效地运行 Docker 和 Linux。Docker Desktop 替代品，可帮助您更快地工作。![Freeware][Freeware Icon]
```
Comment (jaywcjlove) [inline README-zh.md:471]: Why was it removed? This has always been my productivity tool.

@Qifull

File: README-zh.md [hunk 5]
```diff
@@ -234,18 +235,15 @@ Awesome Mac
...
 * [cmus](https://cmus.github.io/) - 命令行播放音乐应用。 [![Open-Source Software][OSS Icon]](https://github.com/cmus) ![Freeware][Freeware Icon]
 * [Dnote](https://www.getdnote.com/) - 命令行上的笔记本，支持多设备同步和网络界面。 [![Open-Source Software][OSS Icon]](https://github.com/dnote/dnote) ![Freeware][Freeware Icon]
 * [Fish Shell](https://fishshell.com/) - 智能且用户友好的命令行终端。 [![Awesome List][awesome-list Icon]](https://github.com/fisherman/awesome-fish-shell#readme)
 * [Glances](https://github.com/nicolargo/glances) - 在命令行中查看你系统运行状态的工具。[![Open-Source Software][OSS Icon] ![Freeware][Freeware Icon]](https://github.com/nicolargo/glances)
 * [httpie](https://httpie.org) - HTTPie 是一个让你微笑的命令行 HTTP 客户端。 [![Open-Source Software][OSS Icon]](https://github.com/jakubroztocil/httpie) ![Freeware][Freeware Icon]
 * [hyper](https://hyper.is) - 基于 Web 技术的终端，直接替代自带的 Terminal。[![Open-Source Software][OSS Icon]](https://github.com/zeit/hyper) ![Freeware][Freeware Icon]
-* [HyperTerm](https://hyperterm.org/) - 一款基于 Node 开发的终端软件，逼格很高。[![Open-Source Software][OSS Icon]](https://github.com/zeit/hyperterm) ![Freeware][Freeware Icon]
```
Comment (jaywcjlove) [inline README-zh.md:246]: - https://github.com/vercel/hyper
- https://hyper.is

This is open-source and still being updated, right? I just need to update the URL, correct?

File: README-zh.md (hunk 4 above)
Comment (Qifull) [reply README-zh.md:471]: Nothing was deleted. I only adjusted the order of the character templates.

File: README-zh.md (hunk 5 above)
Comment (Qifull) [reply README-zh.md:246]: The issue isn’t here; it mainly overlaps with the previous one. You can go ahead and delete it. Just a note: Hyperterm has been renamed to Hyper.

File: .vscode/settings.json
Comment (jaywcjlove) [inline .vscode/settings.json:1]: @Qifull The configuration file is still there.

Comment (Qifull) [reply .vscode/settings.json:1]: OK，Let me figure out.

**Pull request merged**